### DISCUSSION TITLE
Is fragment response mode supported?

### DISCUSSION THREAD

**Discussion by thomasjons** [zitadel/zitadel#5751] [Q&A]
I'm trying out Zitadel with MSAL as the client library but I'm having some issues with the authorization code response.
Even though response_mode is set to fragment in the authorize request, the response still returns the parameters as a query string.

Is there a way to make Zitadel return the response as a fragment instead?

I am using the Load balancing example https://zitadel.com/docs/self-hosting/deploy/loadbalancing-example

**Comment by hifabienne** [comment]
@livio-a Can you help with this question?
